.idea/compiler.xml

```diff
@@ -6,19 +6,20 @@
         <sourceOutputDir name="target/generated-sources/annotations" />
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
-        <module name="platform-qa-ui-answer" />
+        <module name="platform-qa-ui-social" />
```

Comment:
All idea files should not be committed in the sources.

Reply:
I have committed already!! how I can remove the last commit??

Reply:
I did not change me, this change was done automatically I think !!

Reply:
Yes, the changes are done by the editor automatically. You should simply remove the folder `.idea`.

Reply:
ok thx :)